src/istgt_misc.c

```diff
@@ -111,19 +114,34 @@ static int mpool_inited = 0;
 static size_t mpool_minSize = 0;
 static size_t mpool_maxSize = 0;
 
-//uint8_t PoolMap[MAX_SIZE+1];
-//uint32_t PoolSize[20]     = { 4096, 8192, 12288, 0, 0, 0,0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 };
-uint32_t PoolSizeActual[50] = {0} ;
+// uint8_t PoolMap[MAX_SIZE+1];
+// uint32_t PoolSize[20]	= { 4096, 8192, 12288, 0, 0, 0,
+//	0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 };
+uint32_t PoolSizeActual[50] = {0};
 
 uint32_t poolBuckUsage[50] = { 0 };
-//uint32_t PoolSize[20]     =  { 1024, 2048, 3072, 4096, 5000, 6000, 8192, 12288, 16384, 32768, 65536, 131072, 262144, 307200, 524288, 0, 0, 0, 0, 0};
-//uint32_t poolBuckLimit[20] = { 1024, 1024,  1024, 1040,  800,  400,  600,   320,   120,   320,   120,    120,    120,     60,    120, 0, 0, 0, 0, 0};
-uint32_t PoolSize[50]     =  { 32,     64,  128,  256,  512, 1024, 2048, 3072, 4096, 5000, 6000, 8192, 12288, 16384, 32768, 65536, 131072, 262144, 307200, 524288, 590000, 856200, 1048680, 0, 0,
-0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0};
-uint16_t poolBuckLimit[50] = { 700, 700, 400, 700, 4096, 4096, 2080, 1024, 1024,  500,  400,  600,   220,   220,   120,   220,    220,    220,    160,    50,     40, 100, 12, 0, 0,
-0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0};
-
-static int      g_poolCnt = 0;
+// uint32_t PoolSize[20]	=  { 1024, 2048, 3072, 4096, 5000, 6000, 8192,
+//	12288,	16384, 32768, 65536, 131072,
+//	262144, 307200, 524288, 0, 0, 0, 0, 0};
+// uint32_t poolBuckLimit[20]	=  { 1024,	1024,	1024,	1040,	800,
+//	400,	600,	320,	120,	320,	120,	120,	120,
+//	60,	120,	0,	0,	0,	0,	0};
```

Comment:
Instead of the above single line comments, a multi-line comment can be used, similar to:
[https://github.com/openebs/istgt/blob/replication/src/istgt_misc.c#L1](url)